week1/homework/assignments.md

```diff
@@ -3,16 +3,34 @@
 **Assignment 1:**
 
 What is an ARN: `?`
+ARN stands for Amazon Resource Name. ARNs uniquely identify AWS resources.
+An ARN consists of partition, service, region, account id and resource-id.
+
+response from command :
+{
+    "Account": "744286628539", 
+    "UserId": "744286628539", 
+    "Arn": "arn:aws:iam::744286628539:root"
+}
 
 **Assignment 2:**
 
 Response from the command: `?`
+'us-east-1' is a supported AWS Region
+
 
 **Assignment 3:**
 
 Command to upload file with the AWS CLI: `?`
+response from command 'aws s3 ls:'
+2022-10-07 09:43:11 hyf-week1-clibucket
+2022-10-07 09:40:59 hyf-week1-console-kiran
+
+command to upload file to bucket from CLI:
+aws s3 mv hello-world.txt s3://<Bucket Name>
```

Comment:
Alternatively, if you want to copy the local file into S3 instead of moving it:

`aws s3 cp` _Copies a local file or S3 object to another location locally or in S3._
Docs: https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/cp.html